EvolvSwiftSDK/EvolvSwiftSDKTests/Common-Tests/EvolvStoreNewTests.swift

```diff
@@ -116,4 +116,45 @@ class EvolvStoreNewTests: XCTestCase {
         
         XCTAssertEqual(activeKeys.count, 5)
     }
+    
+    func testActiveKeysAreReevaluatedOnContextChange() {
+        let context = EvolvContextImpl(remoteContext: ["location" : "UA",
+                                                       "view" : "home",
+                                                       "signedin" : "yes"],
+                                       localContext: [:])
+        
+        let evolvStore = initializeEvolvStore(with: context)
+        let firstActiveKeys = evolvStore.getActiveKeys()
+        
+        evolvStore.set(key: "signedin", value: "no", local: false)
+        
+        let secondActiveKeys = evolvStore.getActiveKeys()
+        
+        XCTAssertNotEqual(firstActiveKeys, secondActiveKeys)
+        XCTAssertEqual(firstActiveKeys, ["home.cta_text", "home.background", "home", "button_color", "cta_text"])
+        XCTAssertEqual(secondActiveKeys, ["home.cta_text", "home.background", "home"])
+    }
+    
+    func testActiveKeysNotifyOfChangeOnContextChange() {
+        let context = EvolvContextImpl(remoteContext: ["location" : "UA",
+                                                       "view" : "home",
+                                                       "signedin" : "yes"],
+                                       localContext: [:])
+        
+        let evolvStore = initializeEvolvStore(with: context)
+        
+        var receivedActiveKeys = [Set<String>]()
+        
+        evolvStore.activeKeys.sink { activeKeys in
+            receivedActiveKeys.append(activeKeys)
+            
+            if receivedActiveKeys.count == 2 {
```

Comment:
if receivableActiveKeys isn't 2 do we run these tests? If we don't I think we don't pick up test failures

Reply:
That's a valid concern, I added an expectation that is only fulfilled when `receivedActiveKeys.count == 2`. 

Otherwise, the test would fail like this:
![IMAGE 2021-08-30 13:55:43](https://user-images.githubusercontent.com/45521753/131329050-a2d2f615-a82e-437e-9c70-ecbde996d3d6.jpg)